### ISSUE TITLE
Making it responsive

### ISSUE THREAD

**Issue by dhiraj1site** [jbaysolutions/vue2-bootstrap-table#20]
Is there any way to make this table responsive? I know it's on the To-do list, but any places to start to make it responsive by myself?

**Comment by syshex** [comment]
want to give it a go ?

**Comment by dhiraj1site** [comment]
Yes, definitely, can you point me to a specific file that I can start with?

**Comment by syshex** [comment]
Nice!

You should be looking into VueBootstrapTable.vue .  The first section of that file is the Template, which I imagine is where you would want to fiddle with. 

let me know if you need any help to figure out anything or with testing, etc
